.github/workflows/packages.yml

```diff
@@ -0,0 +1,77 @@
+name: release-packages
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.version.outputs.tag }}
+    steps:
+      - name: Version
+        id: version
+        run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\//}
```

Comment:
You can simplify this line and make the following `$(cut -d "v" -f2 <<< "$TAG")` obsolete if you remove the `v` right in this line.

tested with bash 3.2:

```
$ export GITHUB_REF='refs/tags/v1.0.0'
$ echo ${GITHUB_REF/refs\/tags\/v/}
1.0.0
```

Reply:
@thisismana nice! PR is adapted to this!